### ISSUE TITLE
image scaling in window

### ISSUE THREAD

**Issue by JeffreyTome** [pornel/ImageAlpha#32]
Every time I drag a small PNG image into ImageAlpha, it scales up the graphic so it's blurry even though the smooth scaling is set to 1x. If I use the slider bottom right each and every time changing it from 1x to something else and then back to 1x it displays at the correct size I need to view it at.

Please make it so that every time you drag a PNG image in, it displays it at the proper 1x size. 

Thanks.

**Comment by pornel** [comment]
Right. If I remember correctly, it starts with auto scaling fitting the image to the window size. Double-click on the image switches to manual scale.

**Comment by JeffreyTome** [comment]
Can you please set it so that by default it is at the correct size? Having to change it for every single graphic is annoying and I am using a couple thousand images for my iOS app I need PNG images for.

**Comment by julian-tr** [comment]
I agree having to move the slider from 1x and back again every time is annoying (although the double-click shortcut mentioned above is slightly easier).

In addition to the scale option honouring the 1x option between image loads, please also make it remember the "Smooth scaling" option. I prefer Smooth scaling turned off, but every time I reload the app, it has defaulted back to on. Aargh!
